comps/comps-katello-server-rhel7.xml

```diff
@@ -71,6 +71,9 @@
 
       <!-- other katello related packages -->
       <packagereq type="default">rhel8-kickstart-setup</packagereq>
+
+      <!-- Additional plugins dependent on katello -->
+      <packagereq type="default">tfm-rubygem-foreman_inventory_upload</packagereq>
```

Comment:
This should be under thirdparty I think. We already have foreman_scc_manager there which is another plugin dependent on katello.